src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -139,24 +139,24 @@ public class LibraryServiceSettings extends ServiceApiSettings {
 
   private final SimpleCallSettings<CreateShelfRequest, Shelf> createShelfSettings;
   private final SimpleCallSettings<GetShelfRequest, Shelf> getShelfSettings;
-  private final PageStreamingCallSettings<ListShelvesRequest, ListShelvesResponse, ListShelvesPagedResponse> listShelvesSettings;
+  private final PageStreamingCallSettings<ListShelvesRequest, ListShelvesResponse, PagedResponseWrappers.ListShelvesPagedResponse> listShelvesSettings;
```

Comment:
+1. The name is definitely too long. The other plan is to have all the classes under a subfolder.